rust/operator-binary/src/config.rs

```diff
@@ -1,60 +1,58 @@
 use indoc::formatdoc;
+use snafu::{ResultExt, Snafu};
 use stackable_druid_crd::{DruidRole, STACKABLE_TRUST_STORE, STACKABLE_TRUST_STORE_PASSWORD};
+use stackable_operator::memory::MemoryQuantity;
 
-pub fn get_jvm_config(role: &DruidRole, heap_in_mebi: u32) -> String {
-    let common_config = formatdoc! {"
-      -server
-      -Duser.timezone=UTC
-      -Dfile.encoding=UTF-8
-      -Djava.io.tmpdir=/tmp
-      -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager
-      -XX:+UseG1GC
-      -XX:+ExitOnOutOfMemoryError
-      -Djavax.net.ssl.trustStore={STACKABLE_TRUST_STORE}
-      -Djavax.net.ssl.trustStorePassword={STACKABLE_TRUST_STORE_PASSWORD}
-      -Djavax.net.ssl.trustStoreType=pkcs12"};
+#[derive(Snafu, Debug)]
+#[allow(clippy::enum_variant_names)]
```

Comment:
#[allow(clippy::enum_variant_names)] not required

Reply:
If e.g. all your variants end on "Error" / "Failure"

Reply:
Ah okay

Reply:
removed :heavy_check_mark: